prime-router/src/test/kotlin/azure/TokenFunctionTests.kt

```diff
@@ -188,6 +207,7 @@ class TokenFunctionTests {
     @Test
     fun `Test expired key`() {
         settings.senderStore.put(sender.fullName, CovidSender(sender, validScope, jwk))
+        settings.organizationStore.put(organization.name, organization)
```

Comment:
If the sender doesn't have an associated organization the server2server code will now raise an error.